Source/Scene/ModelExperimental/ModelExperimentalSceneGraph.js

```diff
@@ -111,14 +111,17 @@ function initialize(sceneGraph) {
     sceneGraph._forwardAxis
   );
 
-  var rootNode = sceneGraph._modelComponents.scene.nodes[0];
```

Comment:
This set of changes should go in #9708 so this can be unit tested along with everything else.